addons/rose/package.json

```diff
@@ -36,8 +36,7 @@
     "ember-named-blocks-polyfill": "^0.2.4",
     "ember-radio-button": "^2.0.1",
     "ember-uuid": "^2.1.0",
-    "emberx-select": "github:adopted-ember-addons/emberx-select#bc5f774",
-    "ember-named-blocks-polyfill": "^0.2.4"
```

Comment:
@randallmorey the deletion of `ember-named-blocks-polyfill` is unrelated to topic of this PR, but as you can see in the line 36, it is already included, so is a duplicate package.

Reply:
Good catch.  Thank you!